src/themes/material.scss

```diff
@@ -206,12 +206,11 @@
 
 		.progress-linear {
 		  display: block;
-		  position: relative;
+		  position: fixed;
```

Comment:
It's a separate issue I noticed that affected progress indicator scrolling with virtual scroll bar with server side paging. I can open separate issue for this if you'd like. I'm new to this open source thing. First real pull request.